tools/c7n_kube/c7n_kube/actions/replicas.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018-2019 Capital One Services, LLC
```

Comment:
tldr, switch this to 2020 Cloud Custodian Project Authors

Generally copyright in this project is held by the contributor, as our cla is not a copyright transfer ie. its pretty much verbatim to the apache cla. generally not relevant for minor updates to extant files, but for new files we've generally try to hold to the author. However, as part of moving to cncf, we're updating copyright to uniform it in conjunction with discussion with major contributors to line up to how other cncf projects do it.